Lib/uuid.py

```diff
@@ -727,3 +741,43 @@ def uuid5(namespace, name):
 NAMESPACE_URL = UUID('6ba7b811-9dad-11d1-80b4-00c04fd430c8')
 NAMESPACE_OID = UUID('6ba7b812-9dad-11d1-80b4-00c04fd430c8')
 NAMESPACE_X500 = UUID('6ba7b814-9dad-11d1-80b4-00c04fd430c8')
+
+if __name__ == '__main__':
+    """
+    Enables uuid to be called from the command line
```

Comment:
I wonder if we can move this logic into a helper method so that we just call the method here? Most libraries seem to follow this e.g `Lib/timeit.py`